src/app/trainee/revalidation-history.interface.ts

```diff
@@ -0,0 +1,29 @@
+export interface IRevalidationHistory {
+  gmcId: string;
+  traineeName: string;
+  cctDate: Date;
+  programmeMembershipType: string;
+  currentGrade: string;
+  recommendations: IRecommendation[];
+}
+
+export interface IRecommendation {
+  Id: number;
+  recommendation: string;
+  outcome: string;
+  gmcSubDueDate: Date;
+  ActSubDate: Date;
```

Comment:
can `ActSubDate` have a lowercase `a`?

Reply:
For sure: Let me feed back to backend as well. Good spot.